addons/portal/views/portal_templates.xml

```diff
@@ -160,26 +171,31 @@
             <t t-set="my_details" t-value="True"/>
             <div class="o_portal_my_home">
                 <div class="oe_structure" id="oe_structure_portal_my_home_1"/>
-                <h3>Documents</h3>
-                <div class="o_portal_docs list-group">
+                <div class="col-12 col-xl order-xl-1">
+                    <div class="row">
+                        <div class="col-lg-12">
+                            <h3 class="o_page_header mt16 mb4">Overview</h3>
+                        </div>
+                    </div>
+                    <div class="row">
+                        <div class="col-lg-12">
+                            <div class="row mt8 o_portal_overview"></div>
+                        </div>
+                    </div>
```

Comment:
is there not a way to simplify the structure here ?
```
<div class="col-12 col-xl order-xl-1">
    <h3 class="o_page_header mt16 mb4">Overview</h3>
    <div class="row mt8 o_portal_overview"></div>
```
and maybe use flex ?
(i did not tested this. I'm just wondering, seems like there are a lot of div that could be avoided)